src/packages/client/src/generation/TSClient.ts

```diff
@@ -1206,14 +1208,14 @@ export class Enum implements Generatable {
   public toJS() {
     const { type } = this
     return `exports.${type.name} = makeEnum({
-${indent(type.values.map(v => `${v}: '${v}'`).join(',\n'), tab)}
+${indent(type.values.map((v) => `${v}: '${v}'`).join(',\n'), tab)}
```

Comment:
When I was looking at this code, I thought that the indentation was weird

Reply:
I'll run Prettier on all source code later 😃